app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express") ;
+const app = express();
+const { getTopics } = require("./controllers/topics.controllers.js");
+
+app.use(express.json());
+
+
+app.get('/api/topics', getTopics);
+
+app.use((err, req, res, next) => {
+    if (err.code === '22P02') {
+      res.status(400).send({ msg: 'Bad Request' });
+    } 
+    else {
+        next(err);
+    }
+   });
+
+  app.use((err, req, res, next) => {
+  
+    if (err.status && err.msg) {
+    
+        res.status(err.status).send({ msg: err.msg });
+    }
+   
+    });
+app.use((err, req, res, next) => {
```

Comment:
Please don't add 'boiler plate' code for error..  It might just need undoing / heavy editing later. If you can't test for error throwing behaviour for any given task, don't add it.  Leave it for later when you need it, and can build it through TDD!